revolv/payments/migrations/0003_auto_20150103_0323.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def populate_data(apps, schema_editor):
+    PaymentInstrumentType = apps.get_model("payments", "PaymentInstrumentType")
+
+    PaymentInstrumentType.objects.create(name="reinvestment")
+    PaymentInstrumentType.objects.create(name="check")
```

Comment:
So the "check" type is for someone who made a manual donation to a project via check right? We should document somewhere (PaymentInstrumentType docstring?) what kinds of payment instrument types there are and what their functions are